packages/blade/src/components/Button/Button/Button.stories.tsx

```diff
@@ -52,6 +60,8 @@ const Page = (): ReactElement => {
       <Primary />
       <Title>Properties</Title>
       <ArgsTable story={PRIMARY_STORY} />
+      <Description markdown=">Note: `icon` prop accepts an `IconComponent` of Blade which should be used as:" />
```

Comment:
I was adding a note for `icon` usage on `Badge` and thought I'd do the same for `Button`

Reply:
Nice, wish there was a way for storybook to automatically show this correctly in generated code 🤞